pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+addopts = -m 'not profiling' --benchmark-skip -l
+markers =
+    profiling: marks tests that can be profiled
```

Comment:
Just curious: What is this file supposed to do? What are these configurations?

Reply:
you can mark certain tests and then only run those, eg you can pytest -m "profiling" and it then runs only tests that are marked with @pytest.mark.profiling. The standard is (addopts) to only run tests that do not have the profiling marker and are not benchmarking tests.

Reply:
Thanks Dirk!